docs/browser.md

```diff
@@ -0,0 +1,58 @@
+---
+id: browser
+title: Browser
+---
+
+Run Prettier in the browser with the `standalone.js` UMD bundle shipped in the NPM package (starting in version 1.13). The new UMD bundle only formats the code and has no support for config and ignore files, no CLI usage and doesn't load plugins automatically.
+
+### `prettier.format(code, options)`
+
+Differently than the `format` function from the [main API](api.md#prettierformatsource-options), this function does not load plugins automatically, so a `plugins` property is required if you want to load external plugins. This also means the parsers included in the Prettier package won't be loaded automatically, so you need to load them before using.
+
+See [Usage](browser.md#usage) below for examples.
+
+## Usage
+
+### Global:
+
+<!-- prettier-ignore -->
+```html
+<script src="https://unpkg.me/prettier@1.13.0/standalone.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-graphql.js"></script>
+<script type="text/javascript">
+prettier.format("query { }", { parser: "graphql", plugins: prettierPlugins });
```

Comment:
Thinking about it, maybe we could just read from `prettierPlugins` global and the `plugins` for additional plugins in case they are not added to the global?

Reply:
It would only take effect when CJS/AMD is not used, which seems relatively uncommon...

Reply:
There might also be situations where you have multiple instances of prettier configured to run with different plugins, I guess? But 🤷

Reply:
Makes sense!